doc/api/process.markdown

```diff
@@ -424,6 +424,31 @@ process.env.foo = 'bar';
 console.log(process.env.foo);
 ```
 
+Assigning a property on `process.env` will implicitly convert the value
+to a string.
+
+Example:
```

Comment:
I'm personally a fan of explicitly marking it as an example, but either way. It seems to be split pretty evenly in that file.